pom.xml

```diff
@@ -106,6 +107,7 @@ Wisconsin-Madison.</license.copyrightOwners>
 		<dependency>
 			<groupId>org.scijava</groupId>
 			<artifactId>scijava-common</artifactId>
+			<version>2.64.1-SNAPSHOT</version>
```

Comment:
This commit should be labeled WIP since it adds a SNAPSHOT dependency. All such dependencies must be purged from history before they merge to master.